src/fragmentarium/ui/display/Transliteration.tsx

```diff
@@ -143,16 +151,60 @@ function DisplayLine({
   )
 }
 
+function DollarLineAndAtLine({
+  line,
+  container = 'div'
+}: {
+  line: LineContainer
+  container?: string
+}): JSX.Element {
+  if (line.type === 'RulingDollarLine') {
+    const rulingDollarLine = line as RulingDollarLine
+    return React.createElement(
+      container,
+      { className: classNames([`Transliteration__ControlLine`]) },
+      [
+        <hr
+          key={101}
+          className={`Transliteration__${line.type} ${rulingDollarLine.number}`}
+        />
+      ]
+    )
+  } else {
+    return React.createElement(
+      container,
+      { className: classNames([`Transliteration__ControlLine`]) },
```

Comment:
My first choice was to pass `null` as props. The current `className` passed as Props isn't used by CSS

Reply:
Please use the correct element name. This not `ControlLine`.